pkg/scheduler/core/equivalence_cache_test.go

```diff
@@ -656,6 +612,13 @@ func TestInvalidateAllCachedPredicateItemOfNode(t *testing.T) {
 	}
 }
 
+func BenchmarkEquivalenceHash(b *testing.B) {
```

Comment:
Seems we don't need this method any more?

Reply:
Or we can also add a benchmark here.

Reply:
I was trying to re-do the original change without any substantial tweaks, so that the review would be smoother.